StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -20,6 +20,7 @@ public enum PaymentSheetError: Error, LocalizedError {
     case unknown(debugDescription: String)
 
     // MARK: Generic errors
+    case integrationError(debugDescription: String)
     case missingClientSecret
     case invalidClientSecret
```

Comment:
I don't see the value in having specific cases for various integration errors. It's not like anyone would want to switch over or otherwise identify a specific integration error _in code_. I'll discuss reworking PaymentSheetError in our next sync.

Reply:
IIRC, we did this instead of having generic "unknown" cases where we pass in a string to be really sure the errors that we log never contain PII.

Reply:
Yeah I recall that too. Does it really help, though? Nothing stops someone from adding a `case invalidClientSecret(valueWhichCouldContainPII: String)` except code review.

Reply:
Yeah the benefit is debatable, there's nothing totally stopping someone from doing it, just guardrails trying to prevent the accidental addition of PII in error messages by making devs think more about the contents of the error message.